tests/test_sites.py

```diff
@@ -203,12 +204,18 @@ def test_get_courses_for_site(self, course_count):
 
     @pytest.mark.parametrize('ce_count', [0, 1, 2])
     def test_get_course_enrollments_for_site(self, ce_count):
-
+        self.users = [EdlyUserFactory() for i in range(3)]
+        for user in self.users:
```

Comment:
instead of this loop, u can just do the following:
```
self.users = [
            EdlyUserFactory(
                profile__edly_sub_organizations=[self.edly_sub_organization]
            )
        ]
```

Reply:
Done.